solidity/contracts/bridge/BridgeState.sol

```diff
@@ -155,6 +158,71 @@ library BridgeState {
         // spent if it was used as an input of a transaction that have been
         // proven in the Bridge.
         mapping(uint256 => bool) spentMainUTXOs;
+        // Determines how frequently a new wallet creation can be requested.
+        // Value in seconds.
+        uint32 walletCreationPeriod;
+        // The minimum BTC threshold in satoshi that is used to decide about
+        // wallet creation or closing.
+        uint64 walletMinBtcBalance;
+        // The maximum BTC threshold in satoshi that is used to decide about
+        // wallet creation.
+        uint64 walletMaxBtcBalance;
+        // The maximum age of a wallet in seconds, after which the wallet
+        // moving funds process can be requested.
+        uint32 walletMaxAge;
+        // 20-byte wallet public key hash being reference to the currently
+        // active wallet. Can be unset to the zero value under certain
+        // circumstances.
+        bytes20 activeWalletPubKeyHash;
+        // Maps the 20-byte wallet public key hash (computed using Bitcoin
+        // HASH160 over the compressed ECDSA public key) to the basic wallet
+        // information like state and pending redemptions value.
+        mapping(bytes20 => Wallets.Wallet) registeredWallets;
+    }
+
+    event WalletParametersUpdated(
+        uint32 walletCreationPeriod,
+        uint64 walletMinBtcBalance,
+        uint64 walletMaxBtcBalance,
+        uint32 walletMaxAge
+    );
+
+    /// @notice Updates parameters of wallets.
+    /// @param _walletCreationPeriod New value of the wallet creation period
+    /// @param _walletMinBtcBalance New value of the wallet minimum BTC balance
+    /// @param _walletMaxBtcBalance New value of the wallet maximum BTC balance
+    /// @param _walletMaxAge New value of the wallet maximum age
+    /// @dev Requirements:
+    ///      - Wallet minimum BTC balance must be greater than zero
+    ///      - Wallet maximum BTC balance must be greater than the wallet
+    ///        minimum BTC balance
+    function updateWalletParameters(
```

Comment:
What's the reasoning behind that function? I think we can just do the update directly in the Bridge's `updateWalletParameters`. That way we keep `BridgeState.sol` simple, and avoid a level of indirection. On the other hand, if we leave that function here, we should add similar functions to other parameters to keep things consistent.

Reply:
> On the other hand, if we leave that function here, we should add similar functions to other parameters to keep things consistent.

Yes, that's the plan! We have just one function now because updating wallet parameters was already implemented but in a separate PR - when working on governance for Bridge - we will need to add a complete set of `update*` functions. My first thought was that the validation of parameters would look best here, next to the declaration of parameters but we can also revisit it when adding the rest of `update*` functions.

Reply:
Ok then! Having the setters next to the declarations sounds good!